Material_FAB/src/main/ets/components/MainPage/FAB_regular.ets

```diff
@@ -61,7 +59,21 @@ export struct FAB_regular {
     },
   }
 
+  private check_fabSize_type() {
+    if(this.fabSize != 'small' && this.fabSize != 'medium' && this.fabSize != 'large')
```

Comment:
Suggest to define as enum and use them for better readebility.
Refer to,
https://github.com/Applib-OpenHarmony/Material_Ui_Checkbox/pull/1/files#diff-c999b5fc24fe70d53d7a39c368b3fed81cb13cd7dff1427415c224348f9cd4f0